packages/local_auth/local_auth_darwin/CHANGELOG.md

```diff
@@ -1,12 +1,16 @@
+## 2.0.0
+
+- Adds MacOS Support. Requires MacOS 10.15 or higher
+
 ## 1.2.2
 
-* Adds compatibility with `intl` 0.19.0.
+- Adds compatibility with `intl` 0.19.0.
```

Comment:
Please do not reformat the file.

Reply:
Oops, my prettier on my vs code must have done that automatically. Will fix